### PR TITLE
Removed deprecated instaamqp instrumentation from CI pipeline

### PR DESCRIPTION
**Pull request by sanojsubran**
This PR prevents futures releases of instaamqp when we run the CI pipeline based releases.

### CODE REVIEW

**Review by nithinputhenveettil — CHANGES_REQUESTED** [review]
Just added one suggestion, other than that, I'm okay with the changes.

**Comment by nithinputhenveettil** [comment]
@sanojsubran, I have another question. Since as we are now starting to deprecate libraries, why don't we implement the same process in other CI/CD scripts? For instance, in Circle CI builds, SonarQube GitHub Actions, and so on. 

What are your thoughts on this?

**Review by Angith — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -75,9 +75,10 @@ jobs:
         #!/bin/bash
 
         EXCLUDED_DIRS="\/.*\/example"
```
[the remark this answers is not in the document]
Comment (sanojsubran) [reply .github/workflows/release.yml:77]: Yes, We can remove it.

**Comment by sanojsubran** [comment]
> @sanojsubran, I have another question. Since as we are now starting to deprecate libraries, why don't we implement the same process in other CI/CD scripts? For instance, in Circle CI builds, SonarQube GitHub Actions, and so on.
> 
> What are your thoughts on this?

Yes, I agree. We should do that.

**Pull request merged**